lib/modules/manager/terragrunt/modules.ts

```diff
@@ -87,7 +87,10 @@ export function analyseTerragruntModule(
       }
     } else {
       // The packageName should only contain the path to the repository
-      dep.packageName = pathname.replace(/^\//, '').split('//')[0];
+      dep.packageName = pathname
+        .replace(/^\//, '')
+        .replace('.git', '')
+        .split('//')[0];
       dep.registryUrls = [
         protocol === 'https:' ? `https://${host}` : `https://${hostname}`,
```

Comment:
The [hostname](https://developer.mozilla.org/en-US/docs/Web/API/URL/hostname) value does not include any port information, while [host](https://developer.mozilla.org/en-US/docs/Web/API/URL/host) does include the port. 

So in case you specified your source via ssh port 22, it will ignore the port information and use the default https port 443.